packages/amplify-category-geo/src/service-walkthroughs/mapWalkthrough.ts

```diff
@@ -32,73 +38,79 @@ export const createMapWalkthrough = async (
   const currentMapResources = await getGeoServiceMeta(ServiceName.Map);
   if (currentMapResources && Object.keys(currentMapResources).length > 0) {
     parameters.isDefault = await prompter.yesOrNo(
-        defaultResourceQuestion(ServiceName.Map),
-        true
+      defaultResourceQuestion(ServiceName.Map),
+      true,
     );
-  }
-  else {
-      parameters.isDefault = true;
+  } else {
+    parameters.isDefault = true;
   }
 
   return parameters;
 };
 
+/**
+ *
+ */
 export const mapNameWalkthrough = async (context: any): Promise<Partial<MapParameters>> => {
-    let mapName;
-    while(!mapName) {
-        const [shortId] = uuid().split('-');
-        const mapNameInput = await prompter.input(
-            'Provide a name for the Map:',
-            { validate: alphanumeric(), initial: `map${shortId}` }
-        );
-        if (await checkGeoResourceExists(mapNameInput)) {
-            printer.info(`Geo resource ${mapNameInput} already exists. Choose another name.`);
-        }
-        else mapName = mapNameInput;
-    }
-    return { name: mapName };
```

Comment:
Are all these indentation changes intentional?

Reply:
That's all linting done by a pre-commit hook.

Reply:
pushed a change that undos and circumvents the linting as it appears it hasn't been done on this branch.